base/document-download-api-deployment.yaml

```diff
@@ -51,7 +56,24 @@ spec:
               value: '$(NEW_RELIC_MONITOR_MODE)'
             - name: NOTIFY_ENVIRONMENT
               value: '$(ENVIRONMENT)'
-          resources: {}
+          resources:
+            requests:
+              cpu: "100m"
+              memory: "400Mi"
+            limits:
+              cpu: "400m"
+              memory: "800Mi"
+          ports:
+            - containerPort: 7000
+          readinessProbe:
+            httpGet:
+              path: /_status
```

Comment:
Does not call other services https://github.com/cds-snc/notification-document-download-api/blob/master/app/healthcheck.py